rpc/gnmi/mixed-schema.md

```diff
@@ -99,6 +96,35 @@ constrained to the specified origin. Particularly:
   the specified `origin`. To delete contents from multiple origins, a client
   MUST specify multiple paths within the `delete` of the `SetRequest`.
 
+### Special considerations for the use of Origin CLI
+
+The following rules apply to a setRequest containing `origin` CLI.  TODO:
+Special considerations for mixing native schema are currently undefined.
+
+* If a `SetRequest` contains `origin` `cli` and `origin` `openconfig`, the CLI
+  MUST be given precedence over the OpenConfig origin.  This precedence is
+  independent of the order in which origin CLI appears.  This overrides the
+  gNMI specification requirement of ordered path based precedence within a
+  `SetRequest`.  The goal of this precedence is to create deterministic
+  behavior for resolving overlapping configuration.  For setRequests without
+  `origin` CLI, order based precedence still applies.
+* The `update` operation MUST be supported for setRequests containing `origin`
+  `cli` and `origin` `openconfig`.  The `update` operation is compatible with
+  the requirements for `origin` `cli` to take precedence.
+* The `replace` operation for a setRequest containing only `origin` `cli` MUST
+  be treated as a full device configuration replacement.
+* The `replace` operation for a setRequest containing both CLI and OpenConfig,
+  the OpenConfig and CLI SHOULD be supported.  The content of the two origins
+  SHOULD be be merged, with precedence to CLI for any overlapping
+  configuration, and then the replace operation performed.  Since the CLI is
+  interpreted as a full configuration replacement, the result of this
+  setRequest should be a full configuration replacement with the merged
+  content of the CLI and OpenConfig paths.
+* A `replace` operation for `origin` `openconfig`, followed by a `update`
+  operation for `origin` `cli` in a single setRequest SHOULD be supported.
+  The goal is to allow configuration replacement to occur at a sub-tree level
+  using both OpenConfig and the CLI configuration data.
```

Comment:
I would like to highlight that this scenario is the highest priority use case for Google.  IMO, this scenario provides the most flexibility to replace the portions of the configuration needed (include the full configuration if desired), while mixing in CLI to cover gaps.   We have operational use cases where replacement of subtrees is necessary.